### ISSUE TITLE
`signal:*` queries return a 504 Gateway Error

### ISSUE THREAD

**Issue by dshemetov** [cmu-delphi/delphi-epidata#950]
These queries should return the data for all the signals  in each data source, but instead they spin until a 504 Gateway Error

```
https://api.covidcast.cmu.edu/epidata/covidcast/?signal=usa-facts:*&time=day:20220104&geo_type=state&geo_values=ca
https://api.covidcast.cmu.edu/epidata/covidcast/?signal=jhu-csse:*&time=day:20220104&geo_type=state&geo_values=ca
https://api.covidcast.cmu.edu/epidata/covidcast/?signal=safegraph:*&time=day:20220104&geo_type=state&geo_values=ca
```

You can confirm that at least one signal has data for each of these with
```
https://api.covidcast.cmu.edu/epidata/covidcast/?signal=usa-facts:confirmed_cumulative_num&time=day:20220104&geo_type=state&geo_values=ca
https://api.covidcast.cmu.edu/epidata/covidcast/?signal=jhu-csse:confirmed_cumulative_num&time=day:20220104&geo_type=state&geo_values=ca
https://api.covidcast.cmu.edu/epidata/covidcast/?signal=safegraph:bars_visit_num&time=day:20220104&geo_type=state&geo_values=ca
```

Possibly a bug with the `signal:*` handling?

**Comment by dshemetov** [comment]
It doesn't seem like we integration tested this. The SourceSignalPair are tested and correctly return SourceSignalPair(signal='usa-facts', signal=True). It may be an issue in forming the database query.

**Comment by krivard** [comment]
@LeonLu2 is picking this up. Leon -- I just did some checks and it seems the recent v4 improvements have resolved the 504 problem, but we indeed do not have any integration tests for signal wildcard.

The task then is to add a `test_signal_wildcard` integration test similar to [the one we have for `test_location_wildcard`](https://github.com/cmu-delphi/delphi-epidata/blob/a18e879ac3172afb971548101aab46cfc9eb1f40/integrations/server/test_covidcast.py#L208).
